packages/wrangler/src/api/startDevWorker/DevEnv.ts

```diff
@@ -23,6 +23,15 @@ export class DevEnv extends EventEmitter {
 		try {
 			await this.config.set(options, true);
 		} catch (e) {
+			const error =
+				e instanceof Error
+					? e
+					: new Error(
+							typeof e === "string"
+								? e
+								: "An unknown error occurred when starting the worker"
+						);
```

Comment:
Perhaps this could just be:

```suggestion
const error = new Error("An error occurred when starting the server", { cause: e });
```